### ISSUE TITLE
False positive: F401: When quoted annotation is used a type parameter

### ISSUE THREAD

**Issue by clacke** [PyCQA/pyflakes#547]
We have two files:

asdf.py:

```
from typing import TYPE_CHECKING

if TYPE_CHECKING:
    from qwer import Qwer


class Asdf:
    qwer: 'Qwer'
```

asdf2.py

```
from typing import Optional, TYPE_CHECKING

if TYPE_CHECKING:
    from qwer import Qwer


class Asdf:
    qwer: Optional['Qwer']
```

Expected behavior: Both files pass `flake8`.

Actual behavior:

    ./asdf2.py:4:5: F401 'qwer.Qwer' imported but unused

**Comment by sigmavirus24** [comment]
What version of pyflakes?

**Comment by clacke** [comment]
2.1.1. Will try with 2.2.0.

**Comment by clacke** [comment]
Fixed in 2.2.0. Thank you, and sorry for the noise.
